<ui version="4.0" >
 <class>FeedPropertiesDialog</class>
 <widget class="QWidget" name="FeedPropertiesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>212</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Feed Properties</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="updateCheck" >
         <property name="text" >
          <string>U&amp;se a custom update interval</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="sizeConstraint" >
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="updateLabel" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Update &amp;every:</string>
           </property>
           <property name="buddy" >
            <cstring>updateInput</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntNumInput" name="updateInput" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="label" >
            <string/>
           </property>
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="referencePoint" >
            <number>0</number>
           </property>
           <property name="suffix" >
            <string> minutes</string>
           </property>
           <property name="specialValueText" >
            <string>Never</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <widget class="QLabel" name="urlLabel" >
           <property name="text" >
            <string>&amp;URL:</string>
           </property>
           <property name="buddy" >
            <cstring>urlEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="feedTitleLabel" >
           <property name="text" >
            <string>&amp;Title:</string>
           </property>
           <property name="buddy" >
            <cstring>feedTitleEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="KLineEdit" name="feedTitleEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>Display name of RSS column</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="KLineEdit" name="urlEdit" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="archiveTab" >
      <attribute name="title" >
       <string>Ar&amp;chive</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QRadioButton" name="globalDefaultRadio" >
         <property name="text" >
          <string>&amp;Use default settings</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="limitArticleNumberRadio" >
               <property name="text" >
                <string>Limit archi&amp;ve to:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KIntSpinBox" name="maxArticleNumberSpin" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="specialValueText" >
                <string>1 article</string>
               </property>
               <property name="suffix" >
                <string> articles</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="limitArticleAgeRadio" >
               <property name="text" >
                <string>&amp;Delete articles older than:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KIntSpinBox" name="maxArticleAgeSpin" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="specialValueText" >
                <string>1 day</string>
               </property>
               <property name="suffix" >
                <string> days</string>
               </property>
               <property name="value" >
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>99</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="keepAllArticlesRadio" >
         <property name="text" >
          <string>&amp;Keep all articles</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
